### PR TITLE
Update opengarage.markdown, add config flow

### PR DESCRIPTION
**Pull request by Danielhiversen**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36032
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/opengarage.markdown [hunk 1]
```diff
@@ -11,84 +11,14 @@ ha_codeowners:
...
-          description: Access key to control device.
-          required: true
-          default: opendoor
-          type: string
-        name:
-          description: Name to use in the Frontend. If not provided, it will use name configured in device.
-          required: false
-          type: string
-{% endconfiguration %}
+To add OpenGarage to your installation, go to Configuration >> Integrations in the UI and enable the Mill integration.
```
Comment (klaasnicolaas) [inline source/_integrations/opengarage.markdown]:
```suggestion
To add OpenGarage to your installation, go to **Configuration** >> **Integrations** in the UI and enable the Mill integration.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
missing the `ha_config_flow: true` on top.

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

File: source/_integrations/opengarage.markdown [hunk 2]
```diff
@@ -7,88 +7,20 @@ ha_release: 0.44
 ha_domain: opengarage
 ha_codeowners:
   - '@danielhiversen'
+ha_config_flow: true
 ---
 
 The `opengarage` cover platform lets you control the open-source [OpenGarage.io](https://opengarage.io/) device through Home Assistant.
 
-## Configuration
+To add OpenGarage to your installation, go to **Configuration** >> **Integrations** in the UI and enable the Mill integration.
```
Comment (caseyjhol) [inline source/_integrations/opengarage.markdown:15]: Shouldn't this be OpenGarage instead of Mill?

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**